test.js

```diff
@@ -336,6 +336,12 @@ if (process.platform !== 'win32') {
 		const cp = execa('non-executable');
 		await t.throwsAsync(cp);
 	});
+
+	test('execa() rejects with correct error and doesn\'t throw if running non-executable with input', async t => {
+		await t.throwsAsync(() => {
```

Comment:
You should be able to shorten it to:

```js
await t.throwsAsync(execa('non-executable', {input: 'Hey!'}), /EACCESS/);
```

Reply:
But it becomes unreadable then...

Reply:
I don't think this is unreadable. But either way, this is more consistent with the other tests:
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L60
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L67
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L71
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L75
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L79
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L83
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L88
  - https://github.com/sindresorhus/execa/blob/075faf3d6885f8f91eb5945d9c8b2f78ad4f0b59/test.js#L198
  - and so on